internal/installcfg/installcfg.go

```diff
@@ -12,27 +11,19 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-type bmc struct {
-	Address  string `yaml:"address"`
-	Username string `yaml:"username"`
-	Password string `yaml:"password"`
-}
-
 type host struct {
-	Name            string `yaml:"name"`
-	Role            string `yaml:"role"`
-	Bmc             bmc    `yaml:"bmc"`
-	BootMACAddress  string `yaml:"bootMACAddress"`
-	BootMode        string `yaml:"bootMode"`
-	HardwareProfile string `yaml:"hardwareProfile"`
+	Name           string `yaml:"name"`
+	Role           string `yaml:"role"`
+	BootMACAddress string `yaml:"bootMACAddress"`
+	BootMode       string `yaml:"bootMode"`
 }
 
 type baremetal struct {
-	ProvisioningNetworkInterface string `yaml:"provisioningNetworkInterface"`
-	APIVIP                       string `yaml:"apiVIP"`
-	IngressVIP                   string `yaml:"ingressVIP"`
-	DNSVIP                       string `yaml:"dnsVIP"`
-	Hosts                        []host `yaml:"hosts"`
+	ProvisioningNetwork string `yaml:"provisioningNetwork"`
```

Comment:
just to understand, previously we added interface name? or what? and now ip?

Reply:
we had interface name, now we won't need it, we just need to define the type of the provisioning network: "unmanaged", "disabled"